### ISSUE TITLE
url describe does not show URL string for pushed routes

### ISSUE THREAD

**Issue by yangcao77** [openshift/odo#3332]
/kind bug
/area url

## What versions of software are you using?

**Operating System:** 
MacOS

**Output of `odo version`:**  
odo v1.2.2 (297316c73)

## How did you run odo exactly?
create and push a route, and run `odo url describe `

## Actual behavior
```
$ odo url describe odoroute1 
NAME          STATE      URL     PORT
odoroute1     Pushed     ://     8080
```

The URL string is empty when running url describe. 

However, `url list` shows the URL string for the same route
```
$ odo url list
Found the following URLs for component nodejs-nodetest-satq in application app:
NAME          STATE      URL                                             PORT     SECURE
odoroute1     Pushed     http://odoroute1-app-che.9.42.26.107.nip.io     8080     false
```

## Expected behavior
Should show URL string for pushed routes.

**Comment by yangcao77** [comment]
Since I'm working on url describe for routes under experimental mode, it's facing the same issue for experimental mode. I will fix this bug for odo1 as well. 

Fix will be included in https://github.com/openshift/odo/pull/3305

/assign

**Comment by mik-dass** [comment]
@yangcao77 There is already a issue regarding this https://github.com/openshift/odo/issues/3274

**Comment by mik-dass** [comment]
> @mik-dass I saw you have a PR merged for #3274 . But it does not fix the issue, the pushed route URL still displayed as ://, and no matter if the URL is secured or not.

I don't have a merged PR that fixes #3274 . It was just mentioned in my PR https://github.com/openshift/odo/pull/3269#issuecomment-636839161.

> Please confirm that #3274 is the same issue as this one. Thanks.

Yes I think this issue is already a part of #3274, which is still open.

**Comment by yangcao77** [comment]
Dup of #3274, closing this issue
